src/index.py

```diff
@@ -0,0 +1,3 @@
+from game import *
+
+print(start())
```

Comment:
Missing end of line. Configure your IDE to let it add them automatically. 

It would especially improve readability when `cat`-ting your files.